src/components/Chat/Sidebar/Sidebar.jsx

```diff
@@ -1,10 +1,22 @@
-import React, { useState } from "react";
+import React, { useEffect, useState } from "react";
 import { SidebarContainer, SidebarSection, RouteLink, SidebarTitle, UserList, SidebarHeader } from "./SidebarElements";
 import Contact from "../Contact";
 import SearchInputBox from "../../Common/SearchInputBox";
 import { FaAngleDown, FaAngleUp } from "react-icons/fa";
+import { useDispatch, useSelector } from "react-redux";
+import { getConnections } from "../../../features/connections/connectionSlice";
 
 const Sidebar = ({ hideSidebar, onlinePeople, offlinePeople, selectedUserId, setSelectedUserId }) => {
+    const dispatch = useDispatch();
+    const { user } = useSelector((state) => state.auth);
+    const {connections, ...}= useSelector((state) => state.connectionData);
```

Comment:
```suggestion
    const {connections}= useSelector((state) => state.connectionData);
```
@sumanjeet0012 Sorry, I confused you, I meant by three dots if you need to destructure other variables so write them. you don't need those dots.

Reply:
ok, currently i don't need those.

Reply:
ok, so you can remove the dots :)